config/routes.rb

```diff
@@ -10,11 +10,13 @@
     end
   end
 
-  get "/pages/*id" => 'pages#show', as: :page, format: false
+  get '/pages/*id', to: 'pages#show', as: :page, format: false
 
-  resources :business_categories
-  resources :membership_applications, only: [:new, :create, :edit, :update, :index, :show]
-  resources :companies, only: [:new, :create, :edit, :update, :index, :show]
+  scope(path_names: { new: 'ny', edit: 'redigera' }) do
```

Comment:
This should be a separate PR.  It could be part of the I18n work if you want to refer to that PT entry.